api-reference/beta/api/application_list.md

```diff
@@ -42,41 +42,51 @@ Here is an example of the response. Note: The response object shown here may be
 ```http
 HTTP/1.1 200 OK
 Content-type: application/json
-Content-length: 636
+Content-length: 1229
 
 {
   "value": [
-      {
-        "addIns": [
+    {
+      "api": {
+        "acceptedAccessTokenVersion": 99,
+        "publishedPermissionScopes": [
           {
-            "id": "id-value",
-            "type": "type-value",
-            "properties": [
-              {
-                "key": "key-value",
-                "value": "value-value"
-              }
-            ]
-          }
-        ],
-        "appId": "appId-value",
-        "appRoles": [
-          {
-            "allowedMemberTypes": [
-              "allowedMemberTypes-value"
-            ],
-            "description": "description-value",
-            "displayName": "displayName-value",
+            "adminConsentDescription": "adminConsentDescription-value",
+            "adminConsentDisplayName": "adminConsentDisplayName-value",
             "id": "id-value",
             "isEnabled": true,
             "origin": "origin-value",
+            "type": "type-value",
+            "userConsentDescription": "userConsentDescription-value",
+            "userConsentDisplayName": "userConsentDisplayName-value",
             "value": "value-value"
           }
-        ],
-        "availableToOtherOrganizations": true,
-        "displayName": "displayName-value",
-        "errorUrl": "errorUrl-value"
+        ]
+      },
+      "allowPublicClient": true,
+      "applicationAliases": [
+        "applicationAliases-value"
+      ],
+      "appRoles": [
```

Comment:
appRoles should be removed from documentation. This applies to the all references of this property in the docs.